flash/core/model.py

```diff
@@ -156,8 +160,13 @@ def predict(
         output = data_pipeline.uncollate_fn(predictions)  # TODO: pass batch and x
         return output
 
-    def configure_optimizers(self) -> torch.optim.Optimizer:
-        return self.optimizer_cls(filter(lambda p: p.requires_grad, self.parameters()), lr=self.learning_rate)
+    def configure_optimizers(
+            self
+    ) -> Union[Tuple[Tuple[torch.optim.Optimizer], Any], Tuple[torch.optim.Optimizer]]:
+        optimizers = self.optimizer_cls(filter(lambda p: p.requires_grad, self.parameters()), lr=self.learning_rate),
+        if self.schedulers:
+            return optimizers, self.schedulers
```

Comment:
```suggestion
            return optimizers, self.scheduler(optimizer=optimizer)
```

Reply:
this will only work if there is only one optimizer @tchaton ?